### PR TITLE
Feature/accessiblity groups

### PR DESCRIPTION
**Pull request by LcTwisk**
### What's new?
I've made a few changes to the format of the YAML file:

- **Introduce Accessibility group**
Which is a collection of all the available accessibility related values: (`Hint, Label, Value`).

- **Introduce Copy group**
Which contains the actual copy as shown in the app (previously, the only content of the leaf node).
Something to think about: Do we always need the Copy group or is it only required when there's also an Accessibility group? Currently, it's required

- **Remove Plurals group**
With the introduction of the copy & accessibility groups it's always clear where plurals can occur:
    - As a direct child of the **Copy** group
    - As a direct child of `Hint, Label or Value` in the **Accessibility** group

### Keywords
For readability purposes I'd like to propose writing the fixed keywords we're using like (`Copy, Accessibility, Hint, Label, Value, Zero, One, Other`) upper-cased. So they can be easily distinguished from the rest of the key path (and maybe we should even enforce the rest of the keys to be pascal-cased?). Let me know if you can think of other alternatives.
Whatever we decide on should also be applied to the `ios`, `android` and `shared` keywords I'd say.

### Other
While modifying the key normalisation I decided that it might be better to ONLY normalise the languages that are passed into the render command (previously, we were normalising all languages but still only rendering the passed in subset).

### CODE REVIEW

File: src/actions/normalize.js [hunk 1]
```diff
@@ -24,54 +29,65 @@ const normalizeYaml = (data, platforms) => {
...
-        language: key,
-        keyPath: keyPath,
-        translation: value
-      };
+
+    if (isLeafGroup(value)) {
+      const newKeyPath = [...keyPath, key];
+      const entries = languages.map(language => {
+        const accessibilty =
+          groupKeywords.ACCESSIBILITY in value &&
```
Comment (larslockefeer) [inline src/actions/normalize.js]: TIL

File: src/actions/render.js [hunk 2]
```diff
@@ -1,25 +1,31 @@
-const { groupByKey } = require("../utils/arrayUtils");
-const { mapObject } = require("../utils/objectUtils");
+const Mustache = require("mustache");
+const Result = require("../utils/result");
```
Comment (larslockefeer) [inline src/actions/render.js]: In other files, I grouped imports into external dependencies and internal dependencies. Following that logic, `Result` should be separated at least by a newline.

It's probably best to leave this to the linter by using [something like this](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md)?

File: tests/localicipe.yaml [hunk 3]
```diff
@@ -27,31 +31,73 @@ shared:
...
+          en: |-
+            This is a multiline string that may contain <a href="about:blank">links</a>
+            You better watch out!
+          nl: |-
+            Dit is een string van meerdere regels die <a href="about:blank">links</a> kan bevatten.
+            Let maar eens op!
+  Accessible:
+    RegularComponent:
+      COPY:
+        en: "This component with accessiblity support"
```
Comment (larslockefeer) [inline tests/localicipe.yaml]: `This is a component with accessiblity support`

File: tests/localicipe.yaml [hunk 4]
```diff
@@ -27,31 +31,73 @@ shared:
...
+        nl: "Dit is een component met accessiblity support"
+      ACCESSIBILITY:
+        HINT:        
+          en: "This is the hint"
+          nl: "Dit is de hint"
+        LABEL:
+          en: "This is the label"
+          nl: "Dit is de label"
+    PluralComponent:
```
Comment (larslockefeer) [inline tests/localicipe.yaml]: Consider also adding a plural case for `ZERO` for completeness?

**Comment by LcTwisk** [comment]
> Looks good! Shouldn't we update/extend some tests as well?

Yep, just wanted to agree on the naming/casing before taking new snapshots.

File: yarn.lock [hunk 5]
```diff
@@ -1133,7 +1133,7 @@ eslint-module-utils@^2.4.0:
 
 eslint-plugin-import@^2.18.2:
   version "2.18.2"
-  resolved "https://registry.yarnpkg.com/eslint-plugin-import/-/eslint-plugin-import-2.18.2.tgz#02f1180b90b077b33d447a17a2326ceb400aceb6"
+  resolved "https://nexus.global.picnicinternational.com/repository/npm-public/eslint-plugin-import/-/eslint-plugin-import-2.18.2.tgz#02f1180b90b077b33d447a17a2326ceb400aceb6"
```
Comment (larslockefeer) [inline yarn.lock]: Can you still remove this?

**Pull request merged**